ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-service-insertion.j2

```diff
@@ -7,4 +7,48 @@
 {% if router_service_insertion.enabled is arista.avd.defined(true) %}
 
 Router service-insertion is enabled.
+{%     if router_service_insertion.connections is arista.avd.defined %}
+
+## Network Services Information
+{%         for connection in router_service_insertion.connections | arista.avd.natural_sort('name') %}
+{%             if connection.name is arista.avd.defined and connection.interface is arista.avd.defined %}
+{%                 if connection.interface.ethernet is arista.avd.defined %}
+
+| Name | Interface | Next Hop |
+| ---- | --------- | -------- |
+{%                     break %}
+{%                 endif %}
+{%             endif %}
+{%         endfor %}
+{%         for connection in router_service_insertion.connections | arista.avd.natural_sort('name') %}
+{%             if connection.name is arista.avd.defined and connection.interface is arista.avd.defined %}
+{%                 if connection.interface.ethernet is arista.avd.defined %}
+| {{ connection.name }} | {{ connection.interface.ethernet.interface_name }} | {{ connection.interface.ethernet.next_hop_ip_address }} |
+{%                 endif %}
+{%             endif %}
+{%         endfor %}
+{%         for connection in router_service_insertion.connections | arista.avd.natural_sort('name') %}
+{%             if connection.name is arista.avd.defined and connection.interface is arista.avd.defined %}
```

Comment:
Add a title `Connection over tunnels`
cf https://www.arista.com/en/support/toi/eos-4-30-1f/17683-service-insertion

Add new lines as required

Check also that any connection is configured with tunnel (i.e. do not render an empty table)